packages/mui-material-pigment-css/src/Hidden/MaterialPigmentHidden.ts

```diff
@@ -0,0 +1,3 @@
+import Hidden from '@pigment-css/react/Hidden';
```

Comment:
Let's rename this file to only `Hidden.ts`, that's the convention we follow in all projects (to match the directory name).